### ISSUE TITLE
When selecting text in an Outlook 2013 message and when you have selected about 30 lines, of that message, NVDA will   become very slow and will not read out what you are selecting?

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5173]
**Reported by Brendon22 on 2015-06-20 22:05**
When selecting text in an Outlook 2013 message and when you have selected about 30 lines, of that message, NVDA will   become very slow and will not read out what you are selecting?
No error sound is given, but NVDA should not act up?
I do not have the setting, announce formatting changes after the cursor (can cause a lag) Not checked in the document formatting settings.

Can this please be looked into?
Thank you.

**Comment by Adriani90** [comment]
@Brendon22 I am prety sure this has been improved during last years. Could you please test with NVDA 2018.4.1 and report if this is still an issue? Thanks!

**Comment by Adriani90** [comment]
No updates on this issue since years. I am closing this as abandoned. I cannot reproduce it with NVDA 2020.1 and different versions of Outlook.
